### PR TITLE
Season and holiday configuration options

### PR DESCRIPTION
**Pull request by Thorn-Rose**
Adds `season` and `holiday` configuration options, which can be used on a per-server basis to change the season and holiday on respective servers, as well as documentation for said configuration options.  The `parseSeason` and `parseHoliday` functions have also been updated to allow to them take a undefined value for a argument, and `parseHoliday` has additionally been updated to recognize all holidays listed under the `Holiday` enum.

### CODE REVIEW

**Comment by Thorn-Rose** [comment]
Alright, so I added global configuration values for `season` and `holiday`, which should now allow for a configurable fallback for when a specific season or holiday is not picked on a server.

**Comment by BytewaveMLP** [comment]
Not merging yet; there's a build error that breaks stuff (but of course the build system quietly consumes errors so Actions doesn't fail the build. :u)

**Comment by Tschrock** [comment]
I would react with a newspaper but GitHub won't let me so here :newspaper_roll::boom::octocat:

File: src/ts/server/serverActionsManager.ts [hunk 1]
```diff
@@ -59,8 +60,8 @@ export function createServerActionsFactory(
 	const statesCounter = new CounterService<CharacterState>(10 * SECOND);
 	const logChatMessage: LogChat = (client, text, type, ignored, target) => chat(server, client, text, type, ignored, target);
 
-	world.season = SEASON;
-	world.holiday = HOLIDAY;
+	world.season = parseSeason(server.season) || parseSeason(config.season) || SEASON;
```
Comment (BytewaveMLP) [inline src/ts/server/serverActionsManager.ts]: Need to add imports for these functions; the imports from `commands.ts` were removed, but the ones for `common/utils.ts` weren't added.

**Comment by Thorn-Rose** [comment]
Alright, added those imports!

**Comment by BytewaveMLP** [comment]
LGTM!

**Pull request merged**